channel-db/database.go

```diff
@@ -1,4 +1,4 @@
-package database
+package channel_db
 
```

Comment:
Add contracts and logic:
* when you put a new channel, check if min period changed -> delete outdated/inconsistent timestamp if necessary, need to document this
* remove the timestamp if you remove the channel, add a postcondition

Reply:
I have reflected on this for some time and it seems slightly convoluted and arbitrary to me. 

I have an alternative proposal, to allow customization for different usecases: I will add an optional `keep_last_seen` field to the `PUT api/channel` endpoint, so that the user can decide what's best for their case.